R/deploymentTarget.R

```diff
@@ -315,32 +315,6 @@ updateDeployment <- function(previous, appTitle = NULL, envVars = NULL) {
   )
 }
 
-defaultAppName <- function(recordPath, server = NULL) {
```

Comment:
Removed because it wasn't called anywhere, right?

Reply:
Yes, that's right. The PR description noted its removal.